.github/workflows/ci.yml

```
@@ -48,3 +48,94 @@ jobs:
 
       - name: Build Docker image
         run: docker build .
+
+  tests:
+    runs-on: ubuntu-latest
+    needs: []
```

actionlint (1.7.12)

[error] "needs" section should not be empty

(syntax-check)